components/raftstore/src/store/fsm/apply.rs

```diff
@@ -886,8 +889,7 @@ where
 
             let res = match entry.get_entry_type() {
                 EntryType::EntryNormal => self.handle_raft_entry_normal(apply_ctx, &entry),
-                EntryType::EntryConfChange => self.handle_raft_entry_conf_change(apply_ctx, &entry),
-                EntryType::EntryConfChangeV2 => unimplemented!(),
```

Comment:
NOTE: this can break backward compatibility (i.e. old version tikv running on data written by new version tikv), but PD will reject old version tikv join the cluster, so this will not happen.

Reply:
Better comment in code.